testdata/p4_14_samples/switch_20160512/includes/intrinsic.p4

```diff
@@ -41,12 +41,15 @@ metadata ingress_intrinsic_metadata_t intrinsic_metadata;
 #define _ingress_global_tstamp_         intrinsic_metadata.ingress_global_timestamp
 #define modify_field_from_rng(_d, _w)   modify_field_rng_uniform(_d, 0, (1<<(_w))-1)
 
+<<<<<<< 5b10de115548717dd47418bece7b7ca011f0df5f:testdata/p4_14_samples/switch_20160512/includes/intrinsic.p4
```

Comment:
I am guessing this and the similar merge conflict indicator lines should cause compilation of switch.p4 to fail?

Reply:
I certainly missed when we added this syntax to the language spec :-)